v3/src/components/inspector-panel.scss

```diff
@@ -50,3 +50,78 @@
     }
   }
 }
+
+.codap-inspector-palette {
+  background-color: #EEE;
+  border: 0px #444 solid;
+  position: absolute;
+  left: 5px !important;
+  inset: 0px;
+  max-width: 270px;
+  height: 251px;
```

Comment:
```suggestion
```
I think if you eliminate this it will encompass its children -- as is it's not tall enough to do so.